solutions/src/test/java/ru/sberbank/school/task10/ConcurrentScalableThreadPoolTest.java

```diff
@@ -0,0 +1,169 @@
+package ru.sberbank.school.task10;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicInteger;
+
+public class ConcurrentScalableThreadPoolTest {
+    private static final int MIN = 3;
+    private static final int MAX = 7;
+
+    @Test
+    @DisplayName("Наличие исключения, при неправильных аргументах конструктора")
+    void checkIllegalConstructorArgument() {
+        Assertions.assertThrows(IllegalArgumentException.class, () -> new ConcurrentScalableThreadPool(3, 1));
+        Assertions.assertThrows(IllegalArgumentException.class, () -> new ConcurrentScalableThreadPool(0, 1));
+        Assertions.assertThrows(IllegalArgumentException.class, () -> new ConcurrentScalableThreadPool(-5, 3));
+    }
+
+    @Test
+    @DisplayName("Наличие исключения, при попытке остановки до старта")
+    void checkCantStoppedBeforeStart() {
+        ConcurrentScalableThreadPool threadPool = new ConcurrentScalableThreadPool(MIN, MAX);
+        Assertions.assertThrows(IllegalStateException.class, threadPool::stopNow);
+    }
+
+    @Test
+    @DisplayName("Наличие исключения, при попытке добавить задачу до старта")
+    void cantExecuteBeforeStarted() {
+        ConcurrentScalableThreadPool threadPool = new ConcurrentScalableThreadPool(MIN, MAX);
+        AtomicInteger count = new AtomicInteger(0);
+        Assertions.assertThrows(IllegalStateException.class
+                , () -> threadPool.execute(count::incrementAndGet));
```

Comment:
&#39;,&#39; should be on the previous line.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-529819149>*